contracts/solidity/oz/ERC-20-votes/erc20VotesTest.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+contract ERC20VotesTest is ERC20, ERC20Votes {
+    constructor(uint initialMintAmmount) ERC20("MyToken", "MYT") ERC20Permit("MyToken") {
+        _mint(msg.sender, initialMintAmmount);
+    }
+
+    function _afterTokenTransfer(address from, address to, uint256 amount)
+        internal
+        override(ERC20, ERC20Votes)
+    {
+        super._afterTokenTransfer(from, to, amount);
```

Comment:
With the newest OZ v5.x, the `_beforeTokenTransfer` and `_afterTokenTransfer` hooks are deleted. I believe the new hook `_update` is supposed to be used instead. More info here https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v5.0.0